release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -0,0 +1,860 @@
+|GA
+|GA
+|GA
+
+|CSI AWS EFS Driver Operator
+|TP
+|GA
+|GA
+
+|CSI automatic migration
+|TP
+|TP
+|
+
+|CSI inline ephemeral volumes
+|TP
+|TP
+|
+
+|CSI vSphere Driver Operator
+|TP
+|GA
+|GA
+
+|Shared Resource CSI Driver
+|-
+|TP
+|
+
+|Automatic device discovery and provisioning with Local Storage Operator
+|TP
+|TP
+|
+
+|OpenShift Pipelines
+|GA
+|GA
+|GA
+
+|OpenShift GitOps
+|GA
+|GA
+|GA
+
+|OpenShift sandboxed containers
+|TP
+|GA
+|GA
+
+|Vertical Pod Autoscaler
+|GA
+|GA
+|GA
+
+|Cron jobs
+|GA
+|GA
+|GA
+
+|PodDisruptionBudget
+|GA
+|GA
+|GA
+
+|Adding kernel modules to nodes with kvc
+|TP
+|TP
+|TP
+
+|Egress router CNI plug-in
+|GA
+|GA
+|GA
+
+|Scheduler profiles
+|GA
+|GA
+|GA
+
+|Non-preempting priority classes
+|TP
+|TP
+|TP
+
+|Kubernetes NMState Operator
+|TP
+|GA
+|GA
+
+|Assisted Installer
+|TP
+|TP
+|TP
+
+|AWS Security Token Service (STS)
+|GA
+|GA
+|GA
+
+|Kdump
+|TP
+|TP
+|
+
+|OpenShift Serverless
+|GA
+|GA
+|GA
+
+|OpenShift on ARM platforms
+|-
+|GA
+|GA
+
+|Serverless functions
+|TP
+|TP
+|TP
+
+|Data Plane Development Kit (DPDK) support
+|GA
+|GA
+|GA
+
+|Memory Manager feature
+|TP
+|TP
+|
+
+|CNI VRF plug-in
+|GA
+|GA
+|GA
+
+|Cluster Cloud Controller Manager Operator
+|GA
+|GA
+|GA
+
+|Cloud controller manager for Alibaba Cloud
+|-
+|TP
+|
+
+|Cloud controller manager for Amazon Web Services
+|TP
+|TP
+|
+
+|Cloud controller manager for Google Cloud Platform
+|-
+|TP
+|
+
+|Cloud controller manager for IBM Cloud
+|-
+|GA
+|GA
+
+|Cloud controller manager for Microsoft Azure
+|TP
+|TP
+|
+
+|Cloud controller manager for Microsoft Azure Stack Hub
+|GA
+|GA
+|GA
+
+|Cloud controller manager for {rh-openstack-first}
+|TP
+|TP
+|
+
+|Cloud controller manager for VMware vSphere
+|-
+|TP
+|
+
+|Driver Toolkit
+|TP
+|TP
+|TP
+
+|Special Resource Operator (SRO)
+|TP
+|TP
+|
+
+|Simple Content Access
+|TP
+|GA
+|GA
+
+|Node Health Check Operator
+|TP
+|TP
+|
+
+|Network bound disk encryption (Requires Clevis, Tang)
+|GA
+|GA
+|GA
+
+|MetalLB Operator
+|GA
+|GA
+|GA
+
+|CPU manager
+|-
+|TP
+|
+
+|Pod-level bonding for secondary networks
+|-
+|TP
+|
+
+|IPv6 dual stack
+|GA
+|GA
+|GA
+
+|Multicluster console
+|-
+|TP
+|
+
+|Selectable Cluster Inventory
+|-
+|TP
+|
+
+|Hyperthreading-aware CPU manager policy
+|-
+|TP
+|
+
+|Dynamic Plug-ins
+|-
+|TP
+|
+
+|Hybrid Helm Operator
+|-
+|TP
+|
+
+|Alert routing for user-defined projects monitoring
+|-
+|TP
+|
+
+|Disconnected mirroring with the oc-mirror CLI plug-in
+|-
+|TP
+|
+
+|Mount shared entitlements in BuildConfigs in RHEL
+|-
+|TP
+|
+
+|Mount shared secrets in RHEL
+|-
+|GA
+|GA
+
+|Support for {rh-openstack} DCN
+|-
+|TP
+|
+
+|Support for external cloud providers for clusters on {rh-openstack}
+|-
+|TP
+|
+
+|OVS hardware offloading for clusters on {rh-openstack}
+|-
+|TP
+|
+|====
+
+[id="ocp-4-11-known-issues"]
+== Known issues
+
+// TODO: This known issue should carry forward to 4.8 and beyond! This needs some SME/QE review before being updated for 4.9.
```

Comment:
Should 4.9 be changed to 4.11? I'd think it's already been updated for 4.9.

Reply:
She can change it to 4.11 but it needs to be checked whether the KI should be removed or should stay.